app/konnect/dev-portal/access-and-approval/dev-reg.md

```diff
@@ -32,12 +32,20 @@ All Developers must register through the {{site.konnect_short_name}} Dev Portal.
 
 4. Check your email for registration confirmation. 
 
-5. After you have confirmed your email and received approval from your admin, log 
-into the Dev Portal.
-    
-    If your admin has auto-approve enabled, you can log in right after confirming 
-    your email. Otherwise, you _will not be able_ to log in until your admin approves 
-    the request. If you try to log in right after creating the account and receive an 
-    error message, it's likely that the admin hasn't approved your account yet.
+5. After you have confirmed your email, you will be asked to set a password for your account.
+
+    You’ll receive an email to confirm your email address and to set a password via the link inside the email. After you have confirmed your email address and set a password, you’ll be able to log into the Dev Portal once your account admin has approved your request. 
```

Comment:
```suggestion
    After you have confirmed your email address and set a password, you’ll be able to log into the Dev Portal once your account admin has approved your request. 
```
Moving this sentence to previous step, as it sounds out of order.
Normally I'd shy away from future tense, but I think in this case it makes sense, as you might be waiting for approval.